crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -882,6 +891,7 @@ impl RenderAsset for Mesh {
             buffer_info,
             primitive_topology: mesh.primitive_topology(),
             layout: mesh_vertex_buffer_layout,
+            bind_group: None,
```

Comment:
Haven't reviewed the rest of this PR, but I don't think we should put a bind group in GpuMesh. Not every renderer will have a bind group per mesh (mine does not), and even in bevy_pbr not every mesh will need it.

Reply:
Done! I reverted to using `MeshBindGroup` (though I renamed it). Using a `HashMap<HandleId, BindGroup>` now in #8158. Actually the code looks much cleaner with this.